crates/bitwarden-crypto/src/sensitive/sensitive.rs

```diff
@@ -81,6 +94,12 @@ impl From<SensitiveString> for SensitiveVec {
     }
 }
 
+impl<N: ArrayLength<u8>> From<Sensitive<GenericArray<u8, N>>> for SensitiveVec {
```

Comment:
`GenericArray<u8, N>` can be converted into `[u8; N]` without copies by using `.into()` or `.from()` (they just do a `mem::tranmute`), so I don't think this impl is really needed. We can update the checksum function to be like this without this impl:
```rust
    pub(crate) fn generate_checksum(&mut self) {
        if let Some(uri) = &self.uri {
            let uri_hash: SensitiveVec = Sensitive::new(Box::new({
                <[u8; 32]>::from(
                    sha2::Sha256::new()
                        .chain_update(uri.expose().as_bytes())
                        .finalize(),
                )
            }))
            .into();
            self.uri_checksum = Some(uri_hash.encode_base64(STANDARD))
        }
    }
```

Reply:
It depends what we want to encourage, it might be tricky to catch that `from` is safe here when other from/into make copies?